web-api/tests/recommendations.rs

```diff
@@ -89,29 +61,19 @@ async fn interact(client: &Client, personalization_url: &Url) -> Result<(), Erro
     Ok(())
 }
 
-#[derive(Debug, Deserialize, PartialEq)]
+#[derive(Debug, Deserialize, Clone)]
 struct PersonalizedDocumentData {
     id: String,
     score: f32,
-    #[serde(default)]
-    properties: HashMap<String, Value>,
-}
-
-#[derive(Debug, Deserialize, PartialEq)]
-enum PersonalizedDocumentsError {
-    NotEnoughInteractions,
 }
 
-#[derive(Debug, Deserialize, PartialEq)]
-enum RecommendationsResponse {
-    #[serde(rename = "documents")]
-    Documents(Vec<PersonalizedDocumentData>),
-    #[serde(rename = "kind")]
-    Error(PersonalizedDocumentsError),
+#[derive(Debug, Deserialize)]
+struct SemanticSearchResponse {
```

Comment:
:wrench:  We get recommendation responses here not semantic search responses.

(Which have the same format, but the name is misleading.)

Reply:
renamed